app/javascript/application.js

```diff
@@ -0,0 +1,21 @@
+import { Route, Routes, BrowserRouter as Router } from 'react-router-dom';
+import React from 'react';
+import ReactDOM from 'react-dom';
+import { Provider } from 'react-redux';
+import Greeting from './greeting';
+import store from './store';
+
+const App = () => (
```

Comment:
- [ ] As per project requirements, you should rename this compnent from application.js to App.js
![image](https://github.com/sajibarafatsiddiqui/helloappbywebpackrails/assets/62849872/5ce167a6-bd06-4dd9-ab2d-cf7bb5b77cd3)

Reply:
@t-yanik I am astonished by your comment!!! The component name is App and from your image, it is clear also. Why are you messing the component name with the file name?🤦‍♂️